campbot/checkers.py

```diff
@@ -475,3 +477,62 @@ def __call__(self, contrib, old_version, new_version):
             return True, True
 
         return True, old_doc.quality == new_doc.quality
+    
+class MissingAccess():
+    """
+    Route with no access
+    """
+    def __init__(self):
+        self.name = "Accès Manquant"
+        self.fail_marker = emoji("/images/emoji/apple/red_circle.png?v=3", self.name)
+        self.success_marker = ""
```

Comment:
Not mandatory, but always good to see some green on reports 

```suggestion
        self.success_marker = emoji("/images/emoji/apple/white_check_mark.png?v=3", "Accès corrigé")
```